classes.rb

```diff
@@ -0,0 +1,47 @@
+class Person
+  def initialize(id, name, age, parent_permission)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :parent_permission
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? == true
+  end
```

Comment:
- [ ] This method `can_use_service?` should implement - if a person is of age or if they **_have permission_** from parents. 

```ruby
def can_use_services? 
    of_age? || @parent_permission
  end
```
_Notice that it now covers scenarios where a student may not be of age, but has the permission of the parent._

Reply:
well resolved! ✔️